Cargo.toml

```
@@ -15,8 +15,8 @@ edition = "2018"
 metrics = ["opentelemetry/metrics", "opentelemetry-prometheus", "prometheus"]
 
 [dependencies]
-actix-http = { version = "2.0", features = ["compress"] }
-actix-web = { version = "3.0", default-features = false, features = ["compress"] }
+actix-http = { version = "3.0.0-beta.1", features = ["compress"] }
```

Should probably have this as well for symmetry even through there are no default features currently.

```suggestion
actix-http = { version = "3.0.0-beta.1", default-features = false, features = ["compress"] }
```